fourkitchens/sous-drupal-project
# Simplify project setup and dependencies

- Adds theme build scripts
- Adds setup script (ran via composer)
- Enables sous modules that were once part of the profile setup
- Extends lando to include node/npm and relies on lando to run additional npm scripts

Testing:
- Run the following command to setup your new Drupal site - `composer create-project fourkitchens/sous-drupal-project:dev-remove-profile-legacy-modules midcamp --no-interaction --ignore-platform-reqs` <-- This will setup drupal, find-replace `midcamp` (or whatever your project name is) in various commands, setup emulsify using the emulsifyCLI, and install compound as the component library.
- Verify the setup process with the `composer create-project` command runs without errors (note: you might see some npm warnings along the way but this is from `emsulsify-drupal` dependency installation. Everything still works regardless of these warnings)
- Verify the setup process above provides and `admin` username and generated password
- Change directory into project `cd test-sous-project`
- Run `lando sous-builder-install` and verify the module installs and is enabled
- Go to `https://test-sous-project.lndo.site` and verify you see welcome landing page
